web/cobrands/sass/_mixins.scss

```diff
@@ -10,20 +22,18 @@ $direction: 'left' !default;
     $nav_background_colour: $nav_fg_hover !global;
 }
 
-@mixin button-size() {
-    display: inline-block;
-    margin: 0;
-    padding: 0.75em 1em;
+@mixin button-size($mt:$mt, $mr:$mr, $mb:$mb, $ml:$ml) {
```

Comment:
If we’re allowing custom margin values here, should we also allow custom padding values?